homeassistant/components/twinkly/__init__.py

```diff
@@ -1,28 +1,41 @@
 """The twinkly component."""
 
-import twinkly_client
+import asyncio
+
+from aiohttp import ClientError
+from ttls.client import Twinkly
 
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
 
-from .const import CONF_ENTRY_HOST, CONF_ENTRY_ID, DOMAIN
+from .const import CONF_ENTRY_HOST, CONF_ENTRY_ID, DATA_CLIENT, DATA_DEVICE_INFO, DOMAIN
 
 PLATFORMS = [Platform.LIGHT]
 
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up entries from config flow."""
+    hass.data.setdefault(DOMAIN, {})
 
     # We setup the client here so if at some point we add any other entity for this device,
     # we will be able to properly share the connection.
     uuid = entry.data[CONF_ENTRY_ID]
     host = entry.data[CONF_ENTRY_HOST]
 
-    hass.data.setdefault(DOMAIN, {})[uuid] = twinkly_client.TwinklyClient(
-        host, async_get_clientsession(hass)
-    )
+    hass.data[DOMAIN].setdefault(uuid, {})
```

Comment:
Our standard is to use the config entry entry_id as key under the domain key in hass.data.